tests/python/test_dmatrix.py

```diff
@@ -327,7 +327,7 @@ def test_sparse_dmatrix_csr(self):
         nrow = 100
         ncol = 1000
         x = rand(nrow, ncol, density=0.0005, format='csr', random_state=rng)
-        assert x.indices.max() < ncol - 1
+        assert x.indices.max() <= ncol - 1
```

Comment:
```suggestion
        assert x.indices.max() < ncol
```
I think this is slightly more idiomatic.

Reply:
done.